package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-swipeable-views",
```

Comment:
"name": "react-swipeable-views-workspace",

We should revert most of https://github.com/oliviertassinari/react-swipeable-views/commit/f8731c53701e47d3aefca40c83921ea8df34db32#diff-b9cfc7f2cdf78a7f4b91a753d10865a2.